pkg/coremodel/playlist/coremodel.cue

```diff
@@ -22,6 +22,7 @@ seqs: [
 				interval: string | *"5m"
 
 				// The ordered list of items that the playlist will iterate over.
+				// FIXME! This should not be optional, but changing it makes the godegen awkward
```

Comment:
removing the '?' changes the generated go structs so that it has inline structs rather than the referencing the  `PlaylistItem` object.  any ideas?

Reply:
No, this is a known bug. i first noticed it a few months ago, and @joeblubaugh [just encountered it](https://raintank-corp.slack.com/archives/C03BH67DS2J/p1665025260263919), too.

i haven't tracked down exactly where the issue is yet - it could be in either cue stdlib->openapi, or openapi->go. [This comment illustrates](https://github.com/grafana/grafana/pull/56263#issuecomment-1270023581) the basic debugging procedure i've been using to suss that out